resources/compass/charts/cockpit/templates/compass-apps-microfrontend.yaml

```diff
@@ -24,4 +24,30 @@ spec:
     - label: Application Details
       navigationPath: applications/details/:appId
       viewUrl: '/application/:appId'
-      showInNavigation: true
\ No newline at end of file
+      showInNavigation: true
```

Comment:
I know it's not your change, but why is there a `"_compass_ui_"` in line number 18? It's the core-ui group.

Reply:
Changed.